contentful/apps
# chore: add import from component library to aiig

## Purpose

The purpose of this PR is to add the integration-component-library into the AI Image Generator app, for purpose of POC and to begin replacing highly reused components within the apps, with components from the library. 

## Approach

I chose to replace the `Hyperlink` component with the component library `HyperLink` component. I am only changing this one component for now as a proof of the package successfully working before abstracting or changing any other components. 

I am doing this only after testing the package installation and deploy in a lower profile app: AWS Amplify ✅ 

## Testing steps

Ensure the AIIG app renders correctly - all of these changes are on the config page, so best to check that the hyperlinks there work. I tested this locally and all is well 🙌 

## Dependencies and/or References

Component library can be found [here](https://github.com/contentful/integration-component-library) for reference of the package.
